doneill/er-cli-go
# Open EarthRanger mobile db

Open and list tables in an EarthRanger mobile database

**Test:**
- Build and run the app
- Provide path to exported er database
- Confirm db can open

```bash
# command
er open earthranger.db

# response
earthranger.db successfully opened
```

- list db tables

```bash
# command
er open earthranger.db -t

# response
accounts_user
sqlite_sequence
user_profiles
user_subjects
event_type
event_category
events
attachments
sync_states
patrol_types
patrols
patrol_segments
```
